sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/AzureWebMvcConfigurer.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.aad.webapp;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.oauth2.client.OAuth2AuthorizedClientManager;
+import org.springframework.security.oauth2.client.endpoint.DefaultRefreshTokenTokenResponseClient;
+import org.springframework.security.oauth2.client.endpoint.OAuth2AccessTokenResponseClient;
+import org.springframework.security.oauth2.client.endpoint.OAuth2RefreshTokenGrantRequest;
+import org.springframework.security.oauth2.client.web.method.annotation.OAuth2AuthorizedClientArgumentResolver;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+import java.util.List;
+
+/**
+ * Abstract configuration class, used to make RefreshTokenGrantRequestEntityConverter take effect.
+ */
+public abstract class AzureWebMvcConfigurer implements WebMvcConfigurer {
```

Comment:
Can we delete `AzureWebMvcConfigurer` and let `DefaultAzureWebMvcContext` implements `WebMvcConfigurer`?